boards/nrf9160dk/include/periph_conf.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2021 Mesotic SAS
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup    boards_nrf9160dk
+ * @{
+ *
+ * @file
+ * @brief       Peripheral configuration for the nRF9160DK
+ *
+ * @author      Dylan Laduranty <dylan.laduranty@mesotic.com>
+ *
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include "periph_cpu.h"
+
+#include "board.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Timer configuration
+ */
+static const timer_conf_t timer_config[] = {
+    {
+        .dev      = NRF_TIMER0_S,
+        .channels = 3,
+        .bitmode  = TIMER_BITMODE_BITMODE_32Bit,
+        .irqn     = TIMER0_IRQn
+    },
+    {
+        .dev      = NRF_TIMER1_S,
+        .channels = 3,
+        .bitmode  = TIMER_BITMODE_BITMODE_08Bit,
+        .irqn     = TIMER1_IRQn
+    },
+};
+
+#define TIMER_0_ISR         isr_timer0 /**< Timer0 IRQ*/
+#define TIMER_1_ISR         isr_timer1 /**< Timer1 IRQ */
+
+#define TIMER_NUMOF         ARRAY_SIZE(timer_config) /**< Timer configuration NUMOF */
```

Comment:
```suggestion
#define TIMER_NUMOF         ARRAY_SIZE(timer_config) /**< Timer configuration NUMOF */
/** @} */
```

Same for UART

Reply:
I remove them on purpose to try to make doxygen on static-test happy but it doesn't work...
@aabadie Any idea what is wrong with static-test here ?